relaton_iev.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |spec|
   # spec.add_development_dependency "ruby-debug-ide"
   spec.add_development_dependency "equivalent-xml", "~> 0.6"
   spec.add_development_dependency "simplecov"
+  spec.add_development_dependency "uuidtools"
```

Comment:
If you’re using this as code, not specs, you need to add this as a dependency, not a development dependency.